paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -731,6 +722,86 @@ internal class PaymentSheetViewModelTest {
         }
     }
 
+    @Test
+    fun `On inline link payment with save requested, should set with 'requireSaveOnConfirmation' set to 'true'`() =
```

Comment:
this test and the one below look almost identical, just two variables that differ (`customerRequestedSave` and `requiresSaveOnConfirmation`). Can we extract this out so that the shared code is shared? It really helps for understanding what is actually being tested within this code.

Definitely ok to do in a follow up